Sources/Sentry/include/SentryTracer.h

```diff
@@ -18,6 +20,11 @@ static NSTimeInterval const SentryTracerDefaultTimeout = 3.0;
  */
 - (nullable id<SentrySpan>)activeSpanForTracer:(SentryTracer *)tracer;
 
+/**
+ * Report that the tracer has finished.
+ */
+- (void)tracerDidFinish:(SentryTracer *)tracer;
```

Comment:
`m`: We already have `SentryTracer.finishCallback`. I would like to stick to one concept if possible.

Reply:
This was a extension feature that was moved to `SentryTracerDelegate` because `SentryPerformanceTracker` needs to know when the tracer finishes but `SentryTimeToDisplay` also needs to know when SentryTracer finishes and is using `SentryTracer.finishCallback`.

This is why I prefer to implement listeners as collections, no need to refactor code unrelated to the feature later.

Now I rather leave this as it is and refactor in another PR.

Reply:
Yeah, we can do it in a follow up PR.